src/aleph_client/conf.py

```diff
@@ -30,6 +30,12 @@ class Settings(BaseSettings):
     DEFAULT_RUNTIME_ID: str = (
         "bd79839bf96e595a06da5ac0b6ba51dea6f7e2591bb913deccded04d831d29f4"
     )
+    DEFAULT_ROOTFS_ID: str = (
+        "549ec451d9b099cad112d4aaa2c00ac40fb6729a92ff252ff22eef0b5c3cb613"
```

Comment:
Could you add a comment describing what this rootfs is based on? Is it Ubuntu, Debian, something else?

Reply:
On the client's help, it is specified: https://github.com/aleph-im/aleph-client/pull/156/files/67eeb697af6230858c593a5a93e0f88491c6bf8a#diff-0cf3f4fcc940f01848a5832101eb2274d6f19932363e17a25ff32b62d5a59f84R60